app/services/xccdf_tailoring_file.rb

```diff
@@ -51,6 +52,17 @@ def rule_selections_builder(xml)
     end
   end
 
+  def rule_group_selections_builder(xml)
+    @rule_group_ref_ids.each do |rule_group_ref_id|
+      xml[XCCDF].select(idref: rule_group_ref_id, selected: true)
```

Comment:
You'll probably have to play with the parents' default selected groups :thinkig:

Reply:
Ah good point.. I'll play around with scap workbench a little and see how it works in there. 
If a group is unselected because all of it's rule children were unselected, I'm not sure if this is something we'll need to add to the tailoring file 🤔 
Adding the tailored rule groups using `selected="true"` is important because it can effect the reporting results but if we don't add `selected="false"` for rule groups that don't contain any selected rules in the tailoring file, this shouldn't effect reporting

Reply:
I think it should work like this on the frontend though so if a customer unselects all rules for a group, the group should become unselected as well

Reply:
Checking with openscap to see if there would be any issues if we left the de-selected groups out of the tailoring file(groups with `selected="false"`)